### DISCUSSION TITLE
Why should I use Thorium over Supermium on Windows XP?

### DISCUSSION THREAD

**Discussion by MollySpacefield** [Alex313031/thorium-legacy#66] [Q&A]
It's exactly as the title states. Why should I use the beta version of Thorium that now supports XP over the Supermium browser?
https://github.com/Alex313031/thorium-win7/releases/tag/M122.0.6261.158-XP

**Comment by Alex313031** [comment]
@PerKCheddy Different features, and compiler optimizations for extra speed.

See > [PATCHES.md](https://github.com/Alex313031/thorium-legacy/blob/main/infra/PATCHES.md).
